packages/@vue/cli-plugin-e2e-nightwatch/nightwatch.conf.js

```diff
@@ -0,0 +1,360 @@
+        browserName: 'firefox',
+        alwaysMatch: {
+          // Enable this if you encounter unexpected SSL certificate errors in Firefox
+          // acceptInsecureCerts: true,
+          'moz:firefoxOptions': {
+            args: geckoArgs
+          }
+        }
+      },
+      webdriver: {
+        start_process: true,
+        server_path: Services.geckodriver ? Services.geckodriver.path : '',
+        cli_args: [
+          // very verbose geckodriver logs
+          // '-vv'
+        ]
+      }
+    },
+
+    chrome: {
+      desiredCapabilities: {
+        browserName: 'chrome',
+        'goog:chromeOptions': {
+          // More info on Chromedriver: https://sites.google.com/a/chromium.org/chromedriver/
+          //
+          // w3c:false tells Chromedriver to run using the legacy JSONWire protocol (not required in Chrome 78)
+          w3c: true,
+          args: chromeArgs
+        }
+      },
+
+      webdriver: {
+        start_process: true,
+        server_path: Services.chromedriver ? Services.chromedriver.path : '',
+        cli_args: [
+          // --verbose
+        ]
+      }
+    },
+
+    edge: {
+      desiredCapabilities: {
+        browserName: 'MicrosoftEdge',
+        'ms:edgeOptions': {
+          w3c: true,
+          // More info on EdgeDriver: https://docs.microsoft.com/en-us/microsoft-edge/webdriver-chromium/capabilities-edge-options
+          args: [
+            // '--headless'
+          ]
+        }
+      },
+
+      webdriver: {
+        start_process: true,
+        // Download msedgedriver from https://docs.microsoft.com/en-us/microsoft-edge/webdriver-chromium/
+        //  and set the location below:
+        server_path: '',
+        cli_args: [
+          // --verbose
+        ]
+      }
+    },
+
+    /// ///////////////////////////////////////////////////////////////////////////////
+    // Configuration for when using cucumber-js (https://cucumber.io)                |
+    //                                                                               |
+    // It uses the bundled examples inside the nightwatch examples folder; feel free |
+    // to adapt this to your own project needs                                       |
+    /// ///////////////////////////////////////////////////////////////////////////////
+    'cucumber-js': {
+      src_folders: ['examples/cucumber-js/features/step_definitions'],
+
+      test_runner: {
+        // set cucumber as the runner
+        type: 'cucumber',
+
+        // define cucumber specific options
+        options: {
+          // set the feature path
+          feature_path: 'node_modules/nightwatch/examples/cucumber-js/*/*.feature'
+
+          // start the webdriver session automatically (enabled by default)
+          // auto_start_session: true
+
+          // use parallel execution in Cucumber
+          // parallel: 2 // set number of workers to use (can also be defined in the cli as --parallel 2
+        }
+      }
+    },
```

Comment:
Cucumber-js example is not available in nightwatch 1.7, please remove this.

Reply:
Never mind. I didn't read the title correctly. The problem is that the nightwatch version in the package.json isn't updated, the config is fine.